pages/_app.tsx

```diff
@@ -19,7 +19,8 @@ export default function App({
   const router = useRouter()
   const dialog = router.query.dialog
   const dialogRoute = dialogRoutes.find((dr) => dr.name === dialog)
-  console.log(dialogRoute)
+  const theme = useTheme()
+  const isMobileView = useMediaQuery(theme.breakpoints.down('md'))
```

Comment:
The naming won't conflict since they are in different files/functions and thus are in different scopes. Whenever we need to know if we are on mobile view or not, we can use this hook and assign it to any variable we want without conflict. Good thought though!